python/ray/train/jax.py

```diff
@@ -10,8 +10,6 @@
 from ray.train.worker_group import WorkerGroup
 from ray.util import PublicAPI
 
-import jax
```

Comment:
Let's make sure to add a comment explaining why we don't import jax at the top

Reply:
sgtm